[docs] deploy from source, but not found file or dir

**Describe the issue**

In https://datafuse.rs/overview/building-and-running/ , we can deploy from source: 

```
$ git clone https://github.com/datafuselabs/datafuse.git
$ cd datafuse
$ make setup
$ make run
```

but, if we do not build datafuse before running, will get some errors:

```
bash ./scripts/deploy/fusequery-standalone.sh
fuse-store: 未找到进程
fuse-query: 未找到进程
Start FuseStore...
Waiting on fuse-store 10 seconds...
nohup: 把输出追加到 'nohup.out'
nohup: 无法运行命令 'target/debug/fuse-store': 没有那个文件或目录
Start FuseQuery...
Waiting on fuse-query 10 seconds...
nohup: 把输出追加到 'nohup.out'
nohup: 无法运行命令 'target/debug/fuse-query': 没有那个文件或目录
make: *** [Makefile:9：run] 错误 124
```

--- a/Makefile
+++ b/Makefile
@@ -6,6 +6,7 @@ setup:
 	bash ./scripts/setup/dev_setup.sh
 
 run:
+	bash ./scripts/build/build-debug.sh
 	bash ./scripts/deploy/fusequery-standalone.sh
 
 build:

--- a/Makefile
+++ b/Makefile
@@ -5,12 +5,20 @@ TAG ?= latest
 setup:
 	bash ./scripts/setup/dev_setup.sh
 
-run:
-	bash ./scripts/build/build-debug.sh
+run: build
+	bash ./scripts/deploy/fusequery-standalone.sh release
+
+run-debug: build-debug
 	bash ./scripts/deploy/fusequery-standalone.sh
+
+run-release:
+	bash ./scripts/deploy/fusequery-standalone-from-release.sh
 
 build:
 	bash ./scripts/build/build-native.sh
+
+build-debug:
+	bash ./scripts/build/build-debug.sh
 
 profile:
 	bash ./scripts/ci/ci-run-profile.sh

